src/univariate/continuous/tdist.jl

```diff
@@ -15,6 +15,8 @@ params(d)     # Get the parameters, i.e. (ν,)
 dof(d)        # Get the degrees of freedom, i.e. ν
 ```
 
+To create a TDist with a different location and scale, see `LocationScale`.
```

Comment:
```suggestion
To create a TDist with a different location and scale, use `+` and `*`, see `AffineDistribution` for details.
```

Reply:
I agree with your changes. Two problems however:
1) The suggested reference is not good enough:
![image](https://user-images.githubusercontent.com/61620837/149757989-cb5cd92a-56d4-47cf-87c9-895229be9ee9.png)

2) If there is a new section added (see my comment from 3 minutes ago), the reference should probably be to it, and not `AffineDistribution`. Right?

Reply:
People who read the source code probably have a terminal open and can do `?AffineDistribution`

Reply:
Would not help me much...

```julia
julia> using Distributions

(@v1.7) pkg> st Distributions
      Status `C:\Users\densb\.julia\environments\v1.7\Project.toml`
  [31c24e10] Distributions v0.25.37

help?> AffineDistribution
search:

Couldn't find AffineDistribution
Perhaps you meant MatrixDistribution
  No documentation found.

  Binding AffineDistribution does not exist.
```

Reply:
AffineDistribution is not exported because you're not supposed to use the constructor. It's just a fallback, similar to Truncated or Product which one also should not use.

Reply:
Right. So referring to it for documentation is a bad idea, right?